docs/content/manual/release-specific/v1.1.0/rwx_feature.md

```diff
@@ -0,0 +1,322 @@
+2. Attach the PVC to a StatefulSet/Deployment with 2 pods.
+3. Write some data in the pod.
+4. Power down the ShareManager node.
+5. The ShareManager pod should move to another node.
+6. As the instance manager is on the same node and based on the setting `Automatically Delete Workload Pod when The Volume Is Detached Unexpectedly`, the workload should be redeployed and volume should be available to user.
+7. Power down the workload node.
+8. The workload pods should move to another node based on `Pod Deletion Policy When Node is Down` setting.
+9. Once the pods are up, they should get attached to the volume. Verify the data.
+
+# Kill the nfs process in the ShareManager
+1. Create a PVC with RWX mode using longhorn class by selecting the option `read write many`.
+2. Attach the PVC to a StatefulSet/Deployment with 2 pods.
+3. Write some data in the pod.
+4. Kill the NFS server in the ShareManager pod.
+5. The ShareManager pod should go into terminate then restarted by the ShareManager controller. The ganesha service pid can be seen by `cat /var/run/ganesha.pid`.
+6. Volume should continue to accessible.
+
+# Delete the ShareManager CRD.
+1. Create a PVC with RWX mode using longhorn class by selecting the option `read write many`.
+2. Attach the PVC to a StatefulSet/Deployment with 2 pods.
+3. Write some data in the pod.
+4. Delete the ShareManager CRD.
+5. A new ShareManager CRD should be created.
+6. Workloads should be restarted based on the setting `Automatically Delete Workload Pod when The Volume Is Detached Unexpectedly`.
+
+# Delete the ShareManager pod.
+1. Create a PVC with RWX mode using longhorn class by selecting the option `read write many`.
+2. Attach the PVC to a StatefulSet/Deployment with 2 pods.
+3. Write some data in the pod.
+4. Delete the ShareManager pod.
+5. A new ShareManager pod should be immediately created.
+
+# Drain the ShareManager node.
+1. Create a PVC with RWX mode using longhorn class by selecting the option `read write many`.
+2. Attach the PVC to a StatefulSet/Deployment with 2 pods.
+3. Write some data in the pod.
+4. Drain the ShareManager pod node.
+5. The volume should get detached first, then the shareManager pod should move to another node and Volume should get reattached.
+
+# Disk full on the ShareManager node.
+1. Create a PVC with RWX mode using longhorn class by selecting the option `read write many`.
+2. Attach the PVC to a StatefulSet/Deployment with 2 pods.
+3. Write some data in the pod and make the disk almost full.
+4. Verify the RWX volume is not failed.
+5. Verify the creation of snapshot/backup.
+6. Try to write more data, and the it should error out `no space left`.
+
+# Scheduling failure with RWX volume.
+1. Disable 1 node.
+2. Create a StatefulSet/Deployment with 2 pods.
+3. Attach a volume with RWX mode using longhorn class and selecting the option `read write many`.
+4. Verify the RWX volume gets created with degraded state.
```

Comment:
This only happens if creation with limited availability is enabled, otherwise it will fail to schedule.

Reply:
Corrected